src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -70,8 +70,8 @@ public double getAverageScore() {
    */
   public Statistics(List<UserComment> userCommentList, int topNComments) {
     sentimentBucketList = categorizeToBucketList(userCommentList, topNComments);
-    averageScore = getAverageValue(userCommentList, "score");
```

Comment:
Can we do this rounding on the front-end instead of the backend?

The precision with which these results should be displayed feels more closely aligned w/ front-end logic, and nothing about the backend API indicates that scores will be rounded (i.e. the name of the encoded JSON field isn't `roundedAverageScore`.

Reply:
Updated in Molly's PR; back end not changed anymore